.npmrc

```diff
@@ -1,3 +1,3 @@
-; This file is automatically added by @npmcli/template-oss. Do not edit.
-
```

Comment:
This is just to test accessing to a specific registry. This will be removed once it works.